lib/src/cancelable_operation.dart

```diff
@@ -130,26 +130,33 @@ class CancelableOperation<T> {
     return completer.future;
   }
 
-  /// Creates a new cancelable operation to be completed
-  /// when this operation completes or is cancelled.
-  ///
-  /// The [onValue] and [onError] callbacks behave in the same way as
-  /// for [Future.then], and the result of those callbacks is used to complete
-  /// the returned cancelable operation.
-  ///
-  /// If [onCancel] is provided, and the this operation is canceled,
-  /// the [onCancel] callback is called and the returned operation completes
-  /// with the result returned by that call.
-  ///
-  /// If [onCancel] is not provided, and this operation is canceled, then the
-  /// returned operation is also canceled.
-  ///
-  /// If [propagateCancel] is `true` and the returned operation is canceled
-  /// then this operation is canceled. The default is `false`.
+  /// Creates a new cancelable operation to be completed when this operation
+  /// completes normally or as an error, or is cancelled.
+  ///
+  /// If this operation completes normally the [value] is passed to [onValue]
```

Comment:
@lrhn - WDYT about using the term `source operation` or `original operation`? I think `returned operation` is fairly clear. I intended for the consistent usage of `this operation` to convey the "this" that the method is called on, but I'm not sure if that comes across.

Reply:
@jakemac53 - you and I had discussed this briefly and I think you were ok with `this`. Should we consider making it `[this]` or `` `this` ``?

Reply:
I don't feel strongly, not sure what `[this]` would link to and I would only do that if dartdoc understand it and does something sensible, but `` `this` `` sgtm.

Reply:
It looks like Dartdoc will link `[this]` to the class, which is sensible but maybe not useful in this situation.

Analyzer does not have a definition to jump to, though I'd imagine if we consider it important we could make it align with dartdoc.

Reply:
I think just backticks is good then, linking the class isn't very helpful.

Reply:
I'd say no quoting at all. If you use a quoted \`this\`, it refers to an object and should be a noun.
In "this operation", the "this" is not a noun. (I don't know the English word for what it is, though. Preposition?)

And using "this operation" is fine with me,\